dev-packages/browser-integration-tests/suites/replay/canvas/manualSnapshot/test.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { getReplayRecordingContent, shouldSkipReplayTest, waitForReplayRequest } from '../../../../utils/replayHelpers';
+
+sentryTest('can manually snapshot canvas', async ({ getLocalTestUrl, page, browserName }) => {
+  if (shouldSkipReplayTest() || browserName === 'webkit') {
```

Comment:
We should skip these tests for `bundle` (see https://github.com/getsentry/sentry-javascript/blob/006133b48450ba4d50e64c3650fb7f3f4a600025/dev-packages/browser-integration-tests/suites/replay/canvas/records/test.ts#L7) 

We are not including ReplayCanvas in our CDN bundles (e.g. bundles with tracing + replay, or tracing + replay + feedback), instead users will need to include the replay canvas only bundle from the CDN.